Jamulus.pro

```diff
@@ -116,7 +116,7 @@ win32 {
             DEFINES += _STDINT_H # supposed to solve compilation error in systemdeps.h
             INCLUDEPATH += "C:/Program Files/JACK2/include"
             LIBS += "C:/Program Files/JACK2/lib/$${libjackname}"
```

Comment:
@jujudusud , the LIBS value needs to set different when cross compiling (x86/i386 in a 64-bit Win OS). Current an error is generated when the lib is linked as it cannot find the **libjack.lib** in the folder "C:/Program Files/JACK2/lib/", as it is located in the "C:/Program Files (x86)/JACK2/lib/" folder.

Reply:
That's why I said we have to had a win64 target...
Do you know how to fix that ?

Reply:
is it ok with the 506876e4 commit ?